packages/icons/src/trigger-explorer2.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none"><g fill="#6938C0" fill-rule="evenodd" clip-rule="evenodd"><path d="M2.41 7.403a1.2 1.2 0 0 1 1.2-1.2h3.448a1.2 1.2 0 0 1 1.2 1.2v.092h1.435v-.092a1.2 1.2 0 0 1 1.2-1.2h3.426a1.2 1.2 0 0 1 1.2 1.2v1.3a1.2 1.2 0 0 1-1.2 1.2h-3.426a1.2 1.2 0 0 1-1.2-1.2v-.008H8.258v.09a1.2 1.2 0 0 1-1.2 1.2H5.682v1.275c0 .242.075.466.226.626.142.15.419.313.97.313h2.815v-.283a1.2 1.2 0 0 1 1.2-1.2h3.448a1.2 1.2 0 0 1 1.2 1.2v1.382a1.2 1.2 0 0 1-1.2 1.2h-3.448a1.2 1.2 0 0 1-1.196-1.1H6.878c-.803 0-1.424-.247-1.841-.687-.409-.431-.555-.977-.555-1.451V9.986H3.61a1.2 1.2 0 0 1-1.2-1.2V7.403Zm4.648 0H3.61v1.383h3.448V7.403Zm7.26 0h-3.425v1.3h3.426v-1.3Zm-3.425 4.513h3.448v1.382h-3.448v-1.382Z"/><path d="M17.29 2.412a1.8 1.8 0 0 1 1.8 1.8v9.056a.6.6 0 0 1-1.2 0V4.212a.6.6 0 0 0-.6-.6H5.405a.6.6 0 0 0-.6.6v.163a.6.6 0 1 1-1.2 0v-.163a1.8 1.8 0 0 1 1.8-1.8H17.29ZM14.68 18.46a.6.6 0 0 1-.6.6H5.404a1.8 1.8 0 0 1-1.8-1.8v-2.798a.6.6 0 1 1 1.2 0v2.798a.6.6 0 0 0 .6.6h8.675a.6.6 0 0 1 .6.6Z"/><path d="M18.49 12.669a.6.6 0 0 1 .6.6v1.591a4.2 4.2 0 0 1-4.2 4.2h-3.542a.6.6 0 1 1 0-1.2h3.542a3 3 0 0 0 3-3V13.27a.6.6 0 0 1 .6-.6Z"/><path d="M19.871 6.124a1.8 1.8 0 0 1 1.8 1.8v8.326a.6.6 0 1 1-1.2 0V7.924a.6.6 0 0 0-.6-.6H18.49a.6.6 0 1 1 0-1.2h1.381Zm-2.452 14.949a.6.6 0 0 1-.6.6h-8.32a1.8 1.8 0 0 1-1.8-1.8v-1.278a.6.6 0 1 1 1.2 0v1.277a.6.6 0 0 0 .6.6h8.32a.6.6 0 0 1 .6.6Z"/><path d="M21.071 15.65a.6.6 0 0 1 .6.6v1.223a4.2 4.2 0 0 1-4.2 4.2h-3.286a.6.6 0 0 1 0-1.2h3.286a3 3 0 0 0 3-3V16.25a.6.6 0 0 1 .6-.6Z"/></g></svg>
```

Comment:
Can we rename it from explorer1,explorer2 to be able to search similar icons easily